server/app.ts

```diff
@@ -22,6 +22,8 @@ app.use(compression());
 app.use(cookieParser());
 
 app.use('/api', authRouter);
+app.use('/api/user', userRouter);
+
```

Comment:
We  will not use each route here, we will collect them in the index of the routes